src/EnergyPlus/WaterUse.cc

```diff
@@ -1579,23 +1579,23 @@ namespace WaterUse {
 
         for (int WaterEquipNum = 1; WaterEquipNum <= state.dataWaterUse->numWaterEquipment; ++WaterEquipNum) {
             auto &thisWEq = state.dataWaterUse->WaterEquipment(WaterEquipNum);
-            thisWEq.ColdVolFlowRate = thisWEq.ColdMassFlowRate / Psychrometrics::RhoH2O(DataGlobalConstants::InitConvTemp);
-            thisWEq.HotVolFlowRate = thisWEq.HotMassFlowRate / Psychrometrics::RhoH2O(DataGlobalConstants::InitConvTemp);
+            thisWEq.ColdVolFlowRate = thisWEq.ColdMassFlowRate / Psychrometrics::RhoH2O(Constant::InitConvTemp);
+            thisWEq.HotVolFlowRate = thisWEq.HotMassFlowRate / Psychrometrics::RhoH2O(Constant::InitConvTemp);
```

Comment:
I don't know why we use the same temperature to convert hot and cold water to volume.

Reply:
CW/HWInitConvTemp added in July 2016 [de8e993](https://github.com/NREL/EnergyPlus/commit/de8e993aa80df9d2c22ccabf867d56500e550650) updated many places but did not update WaterUse.cc which this source code preceeded this change. At this time, boilers were changed to use CWInitConvTemp to avoid diffs (see what we do to ourselves), since then it was changed, in boilers, to HWInitConvTemp. So it could also have been an oversight in using the correct initialization when these new initialization temps were added. I also missed this one for steam. It might be time to retire the non-descript InitConvTemp.

    Real64 constexpr SteamInitConvTemp = 100.0;             // [deg C], standard init steam vol to mass flow conversion temp